src/lib/map.ts

```diff
@@ -465,9 +465,7 @@ export class OLMap {
       const [lng, lat] = transform(view.getCenter()!, 'EPSG:3857', 'EPSG:4326')
 
       if (handler.onMoved) {
-        const { width, height } = this.mapDom.getBoundingClientRect()
-        const canPlace = await this.canPlaceFactory([width / 2, height / 2])
-        handler.onMoved([lng, lat, range, zoom], canPlace)
+        handler.onMoved([lng, lat, range, zoom], true)
```

Comment:
I am not sure if this is a good idea because we already have `canPlaceFactory` as `true` by default in this pull request[1].

[1] 
https://github.com/tai271828/disfactory-frontend/pull/51/files#diff-9a9849c16cb2ded5bbfb791952e066bcf87ebf727955fe8242f3407818219eddR77

Reply:
Verified. I don't think this change is a must. As a prototype, I will land this pull request and create another one for communication/discussion.

Reply:
The subsequent pull request for discussion with @yanghaochang104  https://github.com/tai271828/disfactory-frontend/pull/52